.github/workflows/ci.yaml

```diff
@@ -0,0 +1,46 @@
+name: Continuos Integration
+
+on:
+  pull_request:
+    branches:
+      - master
+      - release/*
+  push:
+    branches:
+      - master
+      - release/*
+
+jobs:
+  build:
+    name: Build
+    timeout-minutes: 20
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        node_version:
+          - 9.5.0
```

Comment:
this could be updated to 14 maybe

Reply:
@jtakalainen Yes I agree, but this PR is about replacing Travis with GitHub Actions.